### PR TITLE
feat: activeWallets / offerStats #30, #29

### LINKED ISSUE #29: Resolver for offer stats
- kodadot/nft-gallery#3208

### LINKED ISSUE #30: active wallet resolver
should return distinct caller over events

```sql
SELECT COUNT(DISTINCT e.caller)
FROM event e
```

- kodadot/nft-gallery#3208

### PR DESCRIPTION
**Pull request by petersopko**
PR closes #30, closes #29

### CODE REVIEW

**Comment by petersopko** [comment]
@vikiival check whenever free

File: src/server-extension/resolvers/index.ts [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+@Resolver()
+export class HelloResolver {
+  constructor(private tx: () => Promise<EntityManager>) {}
+
+  @Query(() => Hello)
+  async hello(): Promise<Hello> {
+    return new Hello(`Hey, this is you custom API extension`);
+  }
+}
+
```
Comment (vikiival) [inline src/server-extension/resolvers/index.ts:25]:
```suggestion
```

File: src/server-extension/query/offerStats.ts [hunk 2]
```diff
@@ -0,0 +1,6 @@
+export const offerStats = `SELECT status,
```
Comment (vikiival) [inline src/server-extension/query/offerStats.ts:1]: PLS do proper formatting

**Review by vikiival — CHANGES_REQUESTED** [review]
smol stuff

**Review by vikiival — APPROVED** [review]
We should go pickup chemex filters

**Pull request merged**